### PR TITLE
brackets around connection

### PR DESCRIPTION
**Pull request by GeorgesFORTRY**

### CODE REVIEW

**Review by Hacene010 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "author": "",
   "license": "ISC",
   "scripts": {
-    "start": "node index.js",
+    "start": "nodemon index.js",
```
Comment (LoicBrassart) [inline package.json]: That's **exactly** what I ask you **not** to do -_-
If we deploy this, it won't work because `nodemon` won't be installed on the VPS

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Nope!

File: package.json (hunk 1 above)
Comment (GeorgesFORTRY) [reply package.json]: should we put "dev: nodemon index.js" and run with npm run dev?

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**